README.md

```diff
@@ -145,11 +229,10 @@ See `code/z_preprocessing` for some code to convert images from svs into jpg. Th
 # Still not working? Consider the following...
 
 - Ask a pathologist to look at your visualizations.
-- Make your own heuristic for aggregating patch predictions to get WSI classification. Often, a slide thats 20% abnormal and 80% normal should be classified as abnormal.
+- Make your own heuristic for aggregating patch predictions to find WSI classification. Often, a slide thats 20% abnormal and 80% normal should be classified as abnormal.
```

Comment:
"to determine the WSI-level classification"

Reply:
@jasonwei20 Fixed in newest commit.